Sources/IdentityState.swift

```diff
@@ -117,14 +117,14 @@ class IdentityState {
         }
 
         identityProperties.updateCustomerIdentifiers(updateIdentityMap)
-        saveToPersistence(and: createXDMSharedState, using: event)
+        saveToPersistence(and: resolveXDMSharedState)
     }
 
     /// Remove customer identifiers specified in `event` from the current `IdentityMap`.
     /// - Parameters:
     ///   - event: event containing customer identifiers to remove from the current customer identities
-    ///   - createXDMSharedState: function which creates new XDM shared states
-    func removeCustomerIdentifiers(event: Event, createXDMSharedState: ([String: Any], Event) -> Void) {
+    ///   - resolveXDMSharedState: function which resolves pending XDM shared states
+    func removeCustomerIdentifiers(event: Event, resolveXDMSharedState: ([String: Any]) -> Void) {
         guard let identifiersData = event.data else {
             Log.debug(label: IdentityConstants.LOG_TAG, "IdentityState - Failed to remove identifier as no identifiers were found in the event data.")
             return
```

Comment:
Same here and the next `return` statement, resolve the pending state with the current Identity properties.